product_variant_inactive/views/product_variant_view.xml

```diff
@@ -0,0 +1,35 @@
+<odoo>
+   <record model="ir.ui.view" id="view_stock_product_tree">
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="stock.view_stock_product_tree"/>
+        <field name="arch" type="xml">
+            <!-- Add grey color for inactive record.
+                 Variants are mainly used with stock, 
+                 Then we also embedds color attributes of module stock -->
+            <xpath expr="//tree" position="attributes">
+                <attribute name="colors" translation="off">grey:active==False;red:virtual_available&lt;0;blue:virtual_available&gt;=0;black:virtual_available&gt;=0 </attribute>
+            </xpath>
+        </field>
+    </record>
+
+
+    <record model="ir.ui.view" id="detail_field_custom_in_view">
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_product_tree_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//tree" position="inside">
+                <field name="active" invisible ="1"/>
+                <button type="object" name="button_activate" invisible="1"
+                        attrs="{'invisible': [('active','=',True)]}" icon="fa-ban"/>
+                <button type="object" name="button_deactivate" invisible="1"
```

Comment:
@gdgellatly 
We've done this PR but we'll no more use this module for our uses cases in 10 version. It's just a contribution (base on v8 production install) that works but improvement could be done by community. Modules are done to get additional feature over the time, but as it works well, especially in productivity side.

1/ Accidental click: It depends of your users, then decision to put xml confirm which cost a lot in productivity, must be define as fine tuning in usability custom module depending of your customers I think.

2/ Desactivation of many: you're right an additional action in more menu could be better to reach excellence of features

3/ A group in button could be better, yes.

I propose to put 2/ and 3/ in roadmap of the module but merge as it